Draft edits up to schema.org

sohkai opened the pull request:
Part 1 of likely 3 or 4.

[Pretty diff](https://github.com/COALAIP/specs/pull/2/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8)

Mostly rewords and technical corrections, with some removals of out-of-date todos/notes.

sohkai committed:
Draft edits up to schema.org

sohkai commented on README.md:
```diff
@@ -415,25 +410,16 @@ expansion:
-The JSON-LD parser assumes we've defined the correctly named keys for a `Person` and uses
-`http://schema.org/Person` to replace each of our properties with their more detailed schema
-definition URIs. The result is an automatically mapped set of data that uses an already available
-schema. Each key of a given value now points to a left node on a schema ontology. Leaf nodes are
```
> Each key of a given value now points to a left node on a schema ontology.

This didn't make sense to me, so I've changed it to:

> each key points to the predicate node of an RDF triplet

Let me know if I've got something confused.

TimDaub replied:
I think "left node" should have been "leaf node"

TimDaub replied:
I think "each key points to the predicate node of an RDF triplet" is not necessarily _correct_.
After all, a predicate can just be subclassed by a domain-specific RDF ontology, hence pointing to anther part of the ontology.

TimDaub commented on README.md:
```diff
@@ -249,25 +249,24 @@ described in the following section.
-of the Universal Resource Location (URL), to address resources. This makes RDF exceptionally
-interoperable and extensible.
+The [Resource Description Framework](https://www.w3.org/TR/rdf11-concepts/) is a framework for
+describing ontologies. It uses the [Universal Resource Identifier](https://tools.ietf.org/html/rfc3986)
```
`ontologies`.
Can't find the source anymore, but I once read that the semantic web is only defining _one_ ontology, not many.

TimDaub commented on README.md:
```diff
@@ -282,24 +281,20 @@ W3C submission:
+way. Lessig's blog (the *subject*) is licensed (the *predicate*) under a [Creative Commons
+Attribution 3.0](https://creativecommons.org/licenses/by/3.0/) license (the *object*). Provided by
+Creative Commons, ccREL is just one example of a context-providing schema that is built on top of
+RDF. Others, such as the [Web Ontology Language](https://www.w3.org/TR/owl-features/) (OWL), also
```
`Others, such as the Web Ontology Language`

I think the comparison between OWL and ccREL is not accurate.
ccREL is specifiying a certain real-life domain as RDF.
OWL is superset of RDF that allows to define an ontology using RDF.

Meaning, they're two different things.

sohkai replied:
Replaced OWL with ODRL here

TimDaub commented on README.md:
```diff
@@ -361,37 +357,36 @@ application developer were to handle this data, they could rely on the same sche
+To see how `context` achieves this, we need to explain how JSON-LD magically maps our example's
+self-defined keys (`givenName`, `familyName` and `birthDate`) to their matching properties on the
+`Person` schema. If you look at the [`Person` definition](http://schema.org/Person), you'll notice
+that we didn't choose random keys–these keys were already part of the schema definition. Because of
```
"–**these keys** were already part of the schema definition"

Replace with "they"

TimDaub commented on README.md:
```diff
@@ -361,37 +357,36 @@ application developer were to handle this data, they could rely on the same sche
-    1. If this is the case, traverse the schema until a leaf node (the JSON-LD specification calls
-       this an `identifying blank node`) is found
+1. For each of the model's keys, check if they map to any keys provided in the schema
+    1. For each matched key, traverse the schema until the matching URI is found
```
"For each matched key, traverse the schema until the matching URI is found"

We're kind of removing a lot of information in this edit. Compare to:

"If this is the case, traverse the schema until a leaf node (the JSON-LD specification calls this an `identifying blank node`) is found".

Since we're talking about a JSON-LD parser here, I prefer the old version.

TimDaub reviewed (commented):
Good work, lets discuss!

sohkai committed:
PR feedback for draft editing (up to schema.org)

- Replace OWL with ODRL for more relevant example of context-providing schema
- Rephrase JSON-LD context explaination

sohkai commented:
@TimDaub updated for review: https://github.com/COALAIP/specs/pull/2/commits/65cce260d41fa3a08ff99fec86600c305747fd49

[Pretty diff](https://github.com/COALAIP/specs/pull/2/commits/65cce260d41fa3a08ff99fec86600c305747fd49?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8)

sohkai committed:
Small cosmetic/rewording changes for CONTRIBUTING.md and IMPLEMENTATIONS.md
